tests/test_bibliography.py

```diff
@@ -0,0 +1,58 @@
+# coding: utf-8
+import os
+from citeproc import (
+    Citation,
+    CitationItem,
+    CitationStylesBibliography,
+    CitationStylesStyle,
+    formatter,
+    source,
+)
+from unittest import TestCase
+from citeproc.source.bibtex.bibparse import BibTeXParser
+
+template = {
+        "type": "book",
+        "title": "La Rettorica",
+        "author": [{"literal": "Brunetto Latini"}],
+        "issued": {"date-parts": [[1968]]},
+        "editor": [{"literal": "Francesco Maggini"}],
+        "publisher": "Le Monnier",
+        "location": "Firenze",
+    }
+
+def _pp(string):
+    return string.split(" ")[5]
+
+class TestBibliographyGeneration(TestCase):
+    def test_generate(self):
+        expected_ordinals = {
+            "en-US": {0: "1st", 1: "2nd"},
```

Comment:
why not just tuples?
```suggestion
            "en-US": ("1st", "2nd"),
```

and compare them avoiding yet another loop?

Reply:
Because then we couldn't e.g. test `4th` without also testing `3rd`.

Reply:
might be Friday and I am missing it... but where are we testing them now?

Reply:
@yarikoptic @tmorrell 

I added the requested changes to the ordinals tests.